app/assets/javascripts/profile/index.js

```diff
@@ -1,16 +1,18 @@
-//= require bootstrap-editable
-//= require ../vendor/underscore
+//= require jquery
+//= require jquery_ujs
+//= require bootstrap-sprockets
+//= require underscore.js
+//= require x-editable-bootstrap.js
```

Comment:
any reason for the change to bootstrap-editable then back to x-editable-bootstrap?

Reply:
I think this change is gone after the rebase